cli/pull.go

```diff
@@ -170,20 +170,20 @@ func formatProgressInfo(info ctrd.ProgressInfo, isTerminal bool) string {
 		if info.Total > 0.0 {
 			bar = progress.Bar(float64(info.Offset) / float64(info.Total))
 		}
-		return fmt.Sprintf("%s:\t%s\t%40r\t%8.8s/%s\t\n",
+		return fmt.Sprintf("%s:\t%s\t%40e\t%8.8s/%s\t\n",
```

Comment:
I'm not sure if I fix the `unrecognized printf verb 'r' ` issue in the right way. Is that about to print a float number with scientific notation in here? Maybe `%f` is better?

Reply:
I think this part is introduced by @skoo87 
Could you help to check this?